cypress/e2e/awx/resources/hosts.cy.ts

```diff
@@ -40,7 +40,14 @@ describe('host and inventory host', () => {
     createAndEditAndDeleteHost(false, inventory);
   });
 
-  it('can work with groups tab', () => {
+  it('can create, edit, assosiat and disassosiate groups at inventory -> hosts -> groups tab', () => {
```

Comment:
it's missing the `skip`
and let's wait for the discussion about how we should organize those repetitive tests since we already have the `inventoryHost.cy.ts` file